<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MigrateUI</class>
 <widget class="QMainWindow" name="MigrateUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppCafe Migration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SoftwareManager.qrc">
    <normaloff>:/icons/appcafe.png</normaloff>:/icons/appcafe.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="group_migrate">
      <property name="styleSheet">
       <string notr="true">QGroupBox{ background: rgba(30,200,80,100); border-radius: 5px; }</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QPushButton" name="push_start">
         <property name="text">
          <string>Start</string>
         </property>
         <property name="icon">
          <iconset resource="SoftwareManager.qrc">
           <normaloff>:/icons/view-refresh.png</normaloff>:/icons/view-refresh.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Migration to new PBI system is required!!</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="group_reboot">
      <property name="styleSheet">
       <string notr="true">QGroupBox{ background: rgba(250,80,0,150); border-radius: 5px; }</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QPushButton" name="push_reboot">
         <property name="text">
          <string>Restart System</string>
         </property>
         <property name="icon">
          <iconset resource="SoftwareManager.qrc">
           <normaloff>:/icons/app_needroot.png</normaloff>:/icons/app_needroot.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Computer restart is recommended!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="text_progress">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>416</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="SoftwareManager.qrc">
     <normaloff>:/icons/window-close.png</normaloff>:/icons/window-close.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="SoftwareManager.qrc"/>
 </resources>
 <connections/>
</ui>
